charts/spire/charts/spire-server/README.md

```diff
@@ -315,3 +315,16 @@ In order to run Tornjak with simple HTTP Connection only, make sure you don't cr
 | `tests.bash.image.pullPolicy`              | The image pull policy                                                                                                                                                                                                                  | `IfNotPresent`                                                                   |
 | `tests.bash.image.version`                 | This value is deprecated in favor of tag. (Will be removed in a future release)                                                                                                                                                        | `""`                                                                             |
 | `tests.bash.image.tag`                     | Overrides the image tag whose default is the chart appVersion                                                                                                                                                                          | `latest@sha256:60ccbc708bc17803b8253a1c4fdf61e9a68b55e6d0a8ae5735cc276ad4afa2d2` |
+
+### Namespaces
+
+| Name                            | Description                                         | Value          |
+| ------------------------------- | --------------------------------------------------- | -------------- |
+| `namespaces.system.name`        | Name of the Spire system Namespace.                 | `spire-system` |
+| `namespaces.system.create`      | Create a Namespace for Spire system resources.      | `false`        |
+| `namespaces.system.annotations` | Annotations to apply to the Spire system Namespace. | `{}`           |
+| `namespaces.system.labels`      | Labels to apply to the Spire system Namespace.      | `{}`           |
+| `namespaces.server.name`        | Name of the Spire server Namespace.                 | `spire-server` |
+| `namespaces.server.create`      | Create a Namespace for Spire server resources.      | `false`        |
+| `namespaces.server.annotations` | Annotations to apply to the Spire server Namespace. | `{}`           |
+| `namespaces.server.labels`      | Labels to apply to the Spire server Namespace.      | `{}`           |
```

Comment:
Wondering if we should lift the values/templates up to charts/spire/values.yaml and charts/spire/templates/

They aren't really specific to the spire-server chart, and nest the settings deeply when it could just be:
namespaces.system.name right at the root of the values.

Reply:
After further discussion, it probably should be a global under global.spire.namespaces.*

We can use that value space in other places to enable simpler deployment in follow on pr's.